src/js/control-bar/volume-control/volume-control.js

```diff
@@ -22,20 +24,80 @@ class VolumeControl extends Component {
    * @param {Object} [options={}]
    *        The key/value store of player options.
    */
-  constructor(player, options) {
-    super(player, options);
+  constructor(player, options = {}) {
 
-    // hide volume controls when they're not supported by the current tech
-    if (player.tech_ && player.tech_.featuresVolumeControl === false) {
-      this.addClass('vjs-hidden');
+    // Pass the vertical option down to the VolumeBar if
+    // the VolumeBar is turned on.
+    if (typeof options.vertical !== 'undefined' && (!options.volumeBar || isPlain(options.volumeBar))) {
```

Comment:
would be good to default `vertical` to false so it's always a boolean, though, `undefined` will get the results we want as well but being explicit is nice.